Origin: Commit 0cc0531e1a717b24fbf3cbf81fb0b305056cb0a6
Author: Joonas Henriksson <joonas.henriksson@gmail.com>
Last-Update: 2021-01-10
Description: [PATCH] cinnamon: Support version 4.8
 No stylesheet changes should be necessary, so simply symlink to 4.6.
---
 common/cinnamon/4.8 | 1 +
 m4/arc-cinnamon.m4  | 4 ++--
 2 files changed, 3 insertions(+), 2 deletions(-)
 create mode 120000 common/cinnamon/4.8

diff --git a/common/cinnamon/4.8 b/common/cinnamon/4.8
new file mode 120000
index 0000000..be9fc83
--- /dev/null
+++ b/common/cinnamon/4.8
@@ -0,0 +1 @@
+4.6
\ No newline at end of file
diff --git a/m4/arc-cinnamon.m4 b/m4/arc-cinnamon.m4
index d7137b5..2a54840 100644
--- a/m4/arc-cinnamon.m4
+++ b/m4/arc-cinnamon.m4
@@ -36,8 +36,8 @@ AC_DEFUN([ARC_CINNAMON], [
 
     # Evenize the minor version for stable versions
     AS_IF(
-        [test "0$CINNAMON_VERSMJR" -eq 4 -a "0$CINNAMON_VERSMNR" -ge 6], [CINNAMON_VERSION=4.6],
-        [test "0$CINNAMON_VERSMJR" -gt 4], [CINNAMON_VERSION=4.6],
+        [test "0$CINNAMON_VERSMJR" -eq 4 -a "0$CINNAMON_VERSMNR" -ge 8], [CINNAMON_VERSION=4.8],
+        [test "0$CINNAMON_VERSMJR" -gt 4], [CINNAMON_VERSION=4.8],
         [test `expr $CINNAMON_VERSMNR % 2` != "0"],
         [CINNAMON_VERSION="$CINNAMON_VERSMJR.`expr $CINNAMON_VERSMNR + 1`"]
     )
-- 
2.29.2

